### DISCUSSION TITLE
Can I join team?

### DISCUSSION THREAD

**Discussion by aepranata** [ROM-builders/temporary#15476] [General]
I wanna join team sir @Apon77 

My successfull build: https://cirrus-ci.com/task/5478406142296064

My tree: https://github.com/aepranata/android_device_xiaomi_rosy
My telegram: https://t.me/aepranata

My source
Device tree: https://github.com/aepranata/android_device_xiaomi_rosy (Branch: ricedroid)
Kernel tree: https://github.com/aepranata/android_kernel_xiaomi_rosy (Branch: shiro)
Vendor tree: https://github.com/aepranata/android_vendor_xiaomi_rosy (Branch: thirteen)

**Comment by Apon77** [comment]
Please check telegram
